### PR TITLE
12.0 cleanup 1009

### PR DESCRIPTION
**Pull request by yelizariev**

### CODE REVIEW

File: ir_attachment_url/models/binary_fields.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Copyright 2017 Dinar Gabbasov <https://www.it-projects.info/team/GabbasovDinar>
 # Copyright 2018 Rafis Bikbov <https://www.it-projects.info/team/RafiZz>
 # Copyright 2019 Eugene Molotov <https://www.it-projects.info/team/em230418>
+# Copyright 2020 Karl <https://github.com/theangryangel>
```
Comment (yelizariev) [inline ir_attachment_url/models/binary_fields.py:4]: @theangryangel let me know if want me to delete this line

Comment (theangryangel) [reply ir_attachment_url/models/binary_fields.py:4]: I have no problems with it :) I dont really feel like I did much but :man_shrugging:

**Comment by trojikman** [comment]
@yelizariev Are we need this in the 13.0 and beyond

**Review by trojikman — APPROVED** [review]

**Comment by yelizariev** [comment]
I guess yes -- that odoo patch was applied on all versions

**Pull request merged**